misc/x509/src/verifier/der.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// Copyright 2015 Brian Smith.
```

Comment:
It is not really obvious which part is covered by which copyright. Can you not disambiguate this file?

Reply:
This code is taken from [the corresponding file in webpki](https://github.com/briansmith/webpki/blob/0d2699a389182e7490d18df4c0fd99a2c2424a1f/src/der.rs), which I modified for use in libp2p.  As such, I (conservatively) would consider the entire file a derivative work of webpki.